src/iris/api.py

```diff
@@ -3670,6 +3675,18 @@ def on_get(self, req, resp, app_name):
             stats['pct_%s_fail_last_month' % mode] = fail_pct
             stats['pct_%s_other_last_month' % mode] = other_pct
 
+        for mode in cache.modes:
+            cursor.execute('''SELECT COUNT(*) FROM `message`
+                              WHERE `sent` > (CURRENT_DATE - INTERVAL 29 DAY)
+                              AND `sent` < (CURRENT_DATE - INTERVAL 1 DAY)
+                              AND `application_id` = %(application_id)s
+                              AND `mode_id` = (SELECT `id` FROM `mode` WHERE `name` = %(mode)s)''',
+                           {'application_id': app['id'], 'mode': mode})
+            num_sent = cursor.fetchone()
+            if num_sent is not None:
+                num_sent = num_sent[-1]
```

Comment:
1) why `-1` instead of `0`?
2) If no rows are returned, the stats[] entry will be set to None, which might cause problems with things expecting to get ints back for these keys. Should probably set it to 0 if we get back None from the cursor

Reply:
1. -1 is consistent with the earlier stats query, but also it looks like it does that specifically for the median_seconds_to_claim to work. 0 it is.
2. Count(*) should always return 1 row, so I'm not sure if None ever happens. Even if it does happen, I think None would be better than 0, since 0 implies no messages were sent and None implies something crazy happened.